core/services/routing/RouteHandler.php

```diff
@@ -117,6 +141,21 @@ public function handleRoutesForCurrentRequest()
     }
 
 
+    /**
+     * @param RouteInterface $route
+     * @param string         $fqcn
+     * @since $VID:$
+     */
+    private function printDataNodes()
+    {
+        if($this->print_data_nodes) {
+            add_action('admin_footer', [$this->data_node_handler, 'printDataNode']);
+            add_action('wp_footer', [$this->data_node_handler, 'printDataNode']);
+            $this->print_data_nodes = false;
+        }
```

Comment:
this was one of the issues that was happening during activation...
the previous code was trying to print the data node data during activation even though there wasn't any 😐